docs/literate/src/files/getting_started.jl

```diff
@@ -0,0 +1,288 @@
+# ```
+# the pressure.
+# Initial condition consists of initial values for ``\rho``, ``\rho v_1``,
+# ``\rho v_2`` and ``\rho e``.
+# One of the common initial condition for compressible Euler equations is density wave.
+# Let's implement it.
+
+# - Open the downloaded file with notepad or any other text editor.
+# - And go to the 9th line with following code:
+#   ````
+#   initial_condition = initial_condition_weak_blast_wave
+#   ````
+#   Here [`initial_condition_weak_blast_wave`](@ref) is used.
+# - Comment out this line using # symbol:
+#   ````
+#   # initial_condition = initial_condition_weak_blast_wave
+#   ````
+# - Now you can create your own initial conditions. Write following code into a file after the
+#   commented line:
+
+    function initial_condition_density_waves(x,t,equations::CompressibleEulerEquations2D)
+      v1 = 0.1 # velocity along x-axis
+      v2 = 0.2 # velocity along y-axis
+      rho = 1.0 + 0.98 * sin(pi * (sum(x) - t * (v1 + v2))) # density wave profile
+      p = 20 # pressure
+      rho_e = p / (equations.gamma - 1) + 1/2 * rho * (v1^2 + v2^2)
+      return SVector(rho, rho*v1, rho*v2, rho_e)
+    end
+    initial_condition=initial_condition_density_waves
+
+# - Execute following code one more time, but instead of `path_to_file` paste the path to the
+#   `elixir_euler_ec.jl` file from the current folder.
+#   ````
+#   using Trixi
+#   trixi_include(path_to_file)
+#   using Plots
+#   plot(sol)
+#   ````
+# Then you will obtain new solution.
+
+trixi_include(@__MODULE__,joinpath(examples_dir(), "tree_2d_dgsem", "elixir_euler_ec.jl"), #hide #md
+  callbacks=CallbackSet(StepsizeCallback(cfl=1.0)), initial_condition=initial_condition) #hide #md
+plot(sol) #hide #md
+
+# Feel free to add
+# changes into `initial_condition` to observe different solutions.
+
+# Now you are able to download, edit and execute Trixi.jl code.
+
+# ## Next steps
+
+# If you plan on editing Trixi.jl itself, you can download Trixi.jl locally and run it from
+# the cloned directory:
+
+# ### Cloning Trixi.jl
+
+# #### Windows
+
+# If you are using Windows OS, you can clone Trixi.jl directory using a Github Desktop.
+# - If you haven't any github account yet, you have to create it on
+#   the [Github website](https://github.com/join).
+# - Download and install [Github Desktop](https://desktop.github.com/) and then login into
+#   your account.
+# - Open an installed Github Desktop, press `Ctrl+Shift+O`.
+# - In opened window paste `trixi-framework/Trixi.jl` and choose path to the folder, where you want
+#   to save Trixi.jl. Then click `Clone` and Trixi.jl will be cloned to your computer. 
+
+# Now you cloned Trixi.jl and only need to add Trixi.jl packages to Julia.
+# - Open a terminal using `Win+R` and `cmd`. Navigate to the folder with cloned Trixi.jl using `cd`.
+# - Create new directory and start Julia with the `--project` flag set to your local Trixi.jl clone.
+#   ````
+#   mkdir run 
+#   cd run
+#   julia --project=.
+#   ````
```

Comment:
~~~suggestion
# - Create new directory `run`, enter it, and start Julia with the `--project=.` flag:
#   ```shell
#   mkdir run 
#   cd run
#   julia --project=.
#   ```
~~~